task/skopeo-copy/0.1/skopeo-copy.yaml

```diff
@@ -40,7 +41,7 @@ spec:
       default: "true"
   steps:
     - name: skopeo-copy
-      image: quay.io/skopeo/stable:v1.1.1
+      image: quay.io/skopeo/stable:v1.2.0@sha256:6bbf47ed612e7c28c449c57d579565e3defdffc80b58c906186e180a851742e4
```

Comment:
hey @barthy1, I see that v1.3.0 of skopeo has arm support as well so can't we use that tagged image here?

Reply:
sure. However I don't have arm hardware to run the tests.